pyclam/manifold.py

```diff
@@ -115,6 +112,13 @@ def parent(self) -> 'Cluster':
                 raise ValueError(f"root cluster has no parent")
         return self.cache['parent']
 
+    @property
+    def color(self) -> str:
+        """ The color of the cluster node, derived from its LFD """
+        x = 29 + ((19 * 2) * int(round(self.local_fractal_dimension)))
+        y = 206 - ((22 * 2) * int(round(self.local_fractal_dimension)))
```

Comment:
These look like magic lines, should have some comments

Reply:
These numbers need explanation. I would actually prefer if ```color``` were a passable function to the dot-file method. That way this code would live on the side of the user and the user would have complete control over colors for many different applications..